pkg/controllers/jobflow/jobflow_controller_action.go

```diff
@@ -260,10 +260,16 @@ func (jf *jobflowcontroller) loadJobTemplateAndSetJob(jobFlow *v1alpha1flow.JobF
 
 	*job = v1alpha1.Job{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:        jobName,
-			Namespace:   jobFlow.Namespace,
-			Labels:      map[string]string{CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName)},
-			Annotations: map[string]string{CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName)},
+			Name:      jobName,
+			Namespace: jobFlow.Namespace,
+			Labels: map[string]string{
+				CreatedByJobTemplate: GetTemplateString(jobFlow.Namespace, flowName),
+				CreatedByJobFlow:     jobFlow.Name,
```

Comment:
namespace is also needed.

Reply:
I looked at the jobflow controller code and found that the jobs created by jobflow must be under its namespace, so I think namespace should not be needed. cc @hwdef

Reply:
But `CreatedByJobTemplate` contains the namespace, we'd better keep consistent with it.

Reply:
done